e2e/utils/command-utils.ts

```diff
@@ -369,9 +366,7 @@ export function runLernaCLI(
 ): string {
   try {
     const pm = getPackageManagerCommand();
-    const fullCommand = `${pm.runLerna} ${command}${
-      isVerbose() ? ' --verbose' : ''
-    }`;
+    const fullCommand = `${pm.runLerna} ${command}`;
```

Comment:
unsure if this is supposed to be removed or just updated

Reply:
Lerna doesn't have a common overarching `verbose` flag afaik.